discovery-server/src/main/java/app/metatron/discovery/domain/datasource/connection/jdbc/JdbcCSVWriter.java

```diff
@@ -158,7 +160,11 @@ private void writeContents(ResultSet resultSet) throws SQLException, IOException
       // thrown before writing occurs
       objects.clear();
       for( int columnIndex = 1; columnIndex <= numberOfColumns; columnIndex++ ) {
-        objects.add(resultSet.getObject(columnIndex));
+        if(resultSet.getObject(columnIndex) instanceof PrestoArray) {
+          objects.add(resultSet.getString(columnIndex));
+        } else {
+          objects.add(resultSet.getObject(columnIndex));
```

Comment:
Do we really need to use the getObject() method?
Object type will be transformed into a string type within writeRow() method.

I'll test with getString() only and will inform in this pr.

Reply:
It seems to be working well with getString() only.
I've added a related change.